Vagrantfile

```diff
@@ -34,25 +34,23 @@ Vagrant.configure(2) do |config|
     config.vm.box = "ubuntu/vivid64"
     ubuntu_common config
   end
-  config.vm.define "wheezy" do |config|
-    config.vm.box = "debian/wheezy64"
-    deb_common(config)
-  end
+  # Wheezy's backports don't contain Openjdk 8 and the backflips required to
+  # get the sun jdk on there just aren't worth it. We have jessie for testing
+  # debian and it works fine.
```

Comment:
Debian 7 is an officially supported OS for elasticsearch (see https://www.elastic.co/subscriptions/matrix) so I think we should really keep it